src/driftpy/dlob/DLOB.py

```diff
@@ -0,0 +1,834 @@
+import copy
```

Comment:
should we just have the file path be `src/driftpy/dlob/dlob.py` instead of capitalizing, i think we did that in ts because it was a class name convention 🤔